src/Package/MSBuild.VSSetup/files.swr

```diff
@@ -39,7 +39,7 @@ folder InstallDir:\MSBuild\Current\Bin
   file source=$(X86BinPath)BuildXL.Native.dll
   file source=$(X86BinPath)BuildXL.Processes.dll
   file source=$(X86BinPath)BuildXL.Utilities.Core.dll
-  file source=$(X86BinPath)Microsoft.VisualStudio.SolutionPersistence.dll
+  file source=$(X86BinPath)Microsoft.VisualStudio.SolutionPersistence.dll vs.file.ngenApplications="[installDir]\MSBuild\Current\Bin\MSBuild.exe" vs.file.ngenArchitecture=all vs.file.ngenPriority=1
```

Comment:
I may want to change this like #11183, can we wait until I look into that?

Reply:
That other PR looks fine so we can do it here:

```suggestion
  file source=$(X86BinPath)Microsoft.VisualStudio.SolutionPersistence.dll vs.file.ngenApplications="[installDir]\MSBuild\Current\Bin\amd64\MSBuild.exe" vs.file.ngenArchitecture=all vs.file.ngenPriority=1
```

I'm not sure about `ngenPriority` because I don't expect this codepath to be used for VS scenarios so a bit of JITting if you kick off a CLI build _immediately_ after install might be ok, but it's also common to build a `.sln` so it's probably best to avoid JITting? In any case we do it for other DLLs so this is following the pattern.